src/masternodes/rpc_tokens.cpp

```diff
@@ -639,45 +639,77 @@ UniValue getcustomtx(const JSONRPCRequest& request)
 UniValue minttokens(const JSONRPCRequest& request) {
     auto pwallet = GetWallet(request);
 
-    RPCHelpMan{"minttokens",
-               "\nCreates (and submits to local node and network) a transaction minting your token (for accounts and/or UTXOs). \n"
-               "The second optional argument (may be empty array) is an array of specific UTXOs to spend. One of UTXO's must belong to the token's owner (collateral) address" +
-               HelpRequiringPassphrase(pwallet) + "\n",
-               {
-                    {"amounts", RPCArg::Type::STR, RPCArg::Optional::NO,
-                        "Amount as json string, or array. Example: '[ \"amount@token\" ]'"
-                    },
-                    {"inputs", RPCArg::Type::ARR, RPCArg::Optional::OMITTED_NAMED_ARG,
-                        "A json array of json objects. Provide it if you want to spent specific UTXOs",
+    RPCHelpMan{
+        "minttokens",
+        "\nCreates (and submits to local node and network) a transaction minting your token (for accounts and/or "
+        "UTXOs). \n"
+        "The second optional argument (may be empty array) is an array of specific UTXOs to spend. One of UTXO's must "
+        "belong to the token's owner (collateral) address" +
+            HelpRequiringPassphrase(pwallet) + "\n",
+        {
+                          {"amounts",
```

Comment:
The formatting here does not look correct to me.

Reply:
Fixed: [55850b2](https://github.com/DeFiCh/ain/pull/1650/commits/55850b2a298c4d928c2940e752814ed2fa972a9d)